### DISCUSSION TITLE
Next export in 13

### DISCUSSION THREAD

**Discussion by mapsgeek** [vercel/next.js#42220] [Help] [answered]
Will next export still work in next 13? i tried to experiment it with `app` but it turns into error
```
PageNotFoundError: Cannot find module for page: /
    at Object.getPagePath
```
not sure if implementation is wrong/ a bug or it won't be supported anymore since there's no `getStaticPaths`

**Comment by icyJoseph** [comment, marked as answer]
It is in the planned features, see here: https://beta.nextjs.org/docs/app-directory-roadmap#planned-features

> The app directory is in beta and many features are not yet supported. During the beta period, APIs could change based on community feedback. The following features are not yet supported:

And in that list we see `next export`

**Reply by mapsgeek** [reply]
Oh ok thanks, my bad.
